job_bundles/tile_render_with_maya_vray/template.yaml

```
@@ -0,0 +1,271 @@
+      groupLabel: Tile Rendering Settings
+    minValue: 1
+    description: The number of tiles across the y axis.
+    default: 3
+
+steps:
+  - name: Render Regions
+    parameterSpace:
+      taskParameterDefinitions:
+        - name: Frame
+          type: INT
+          range: '{{Param.Frames}}'
+        - name: TileNumberX
+          type: INT
+          range: 1-{{Param.NumXTiles}}
+        - name: TileNumberY
+          type: INT
+          range: 1-{{Param.NumYTiles}}
+    stepEnvironments:
+      - name: Maya
+        description: Runs Maya in the background.
+        script:
+          embeddedFiles:
+            - name: initData
+              filename: init-data.yaml
+              type: TEXT
+              data: |
+                renderer: vray
+                render_layer: '{{Param.RenderLayer}}'
+                scene_file: '{{Param.MayaSceneFile}}'
+                camera: '{{Param.Camera}}'
+                project_path: '{{Param.ProjectPath}}'
+                output_file_path: '{{Param.OutputFilePath}}'
+                render_setup_include_lights: {{Param.RenderSetupIncludeLights}}
+                strict_error_checking: {{Param.StrictErrorChecking}}
+                image_width: {{Param.ImageWidth}}
+                image_height: {{Param.ImageHeight}}
+          actions:
+            onEnter:
+              command: maya-openjd
+              args:
+                - daemon
+                - start
+                - '--path-mapping-rules'
+                - file://{{Session.PathMappingRulesFile}}
+                - '--connection-file'
+                - '{{Session.WorkingDirectory}}/connection.json'
+                - '--init-data'
+                - file://{{Env.File.initData}}
+              cancelation:
+                mode: NOTIFY_THEN_TERMINATE
+            onExit:
+              command: maya-openjd
+              args:
+                - daemon
+                - stop
+                - '--connection-file'
+                - '{{ Session.WorkingDirectory }}/connection.json'
+              cancelation:
+                mode: NOTIFY_THEN_TERMINATE
+    script:
+      embeddedFiles:
+        - name: RenderTile
+          filename: render-tile.py
+          type: TEXT
+          data: |
+            import subprocess
+
+            # Tile num uses 1 based indexing. First tile (top left) is x=1, y=1
+            tileNumX = {{Task.Param.TileNumberX}}
+            tileNumY = {{Task.Param.TileNumberY}}
+            numXTiles = {{Param.NumXTiles}}
+            numYTiles = {{Param.NumYTiles}}
+
+            deltaX, widthRemainder = divmod({{Param.ImageWidth}}, numXTiles)
+            deltaY, heightRemainder = divmod({{Param.ImageHeight}}, numYTiles)
+
+            # Calculate the bounds for the tile. Two adjacent tiles could have
+            # "minX=0, maxX=63, minY=0, maxY=63" and then "minX=64, maxX=127, minY=0, maxY=63"
+            minX = deltaX * (tileNumX - 1)
+            maxX = (deltaX * tileNumX)l

```

```suggestion
            maxX = (deltaX * tileNumX)
```

I think this would fail the script

---

Fixed!

---

Looks like it's still here, I think this commit added it back in: https://github.com/aws-deadline/deadline-cloud-samples/pull/142/commits/cae043489bd3943d5d01d8f2009759092f8c0e8b